frontend/app/views/info/supporterInternational.scala.html

```diff
@@ -0,0 +1,72 @@
+@import com.gu.i18n.CountryGroup
+@import com.gu.memsub.Current
+@import com.gu.salesforce.Tier.Supporter
+@import com.gu.membership.PaidMembershipPlans
+@import views.support.PageInfo
+@(supporterPlans: PaidMembershipPlans[Current, Supporter],
+    pageInfo: PageInfo,
+    pageImages: Seq[model.ResponsiveImageGroup])(implicit token: play.filters.csrf.CSRF.Token, countryGroup: CountryGroup)
```

Comment:
Why do we need to pass a `CountryGroup`, if this page is specific to `RestOfTheWorld` ? It might as well be hardcoded